jaegertracing/jaegertracing.go

```diff
@@ -128,82 +129,101 @@ func TraceWithConfig(config TraceConfig) echo.MiddlewareFunc {
 
 			req := c.Request()
 			opname := "HTTP " + req.Method + " URL: " + c.Path()
+			realIP := c.RealIP()
+			requestID := getRequestID(c) // request-id generated by reverse-proxy
```

Comment:
Is a request ID required for jaeger tracing?
It looks like it would be compatible with the request ID middleware here (same as below)

Reply:
Request id logged into opentracing tags below:
```go
sp.SetTag("request_id", requestID)
```
It is much easier than write custom `RequestIDHandler` for request-id middleware.